### ISSUE TITLE
web: Error loading module 'cms'. Its dependency 'androidx-runtime' was not found.

### ISSUE THREAD

**Issue by eymar** [JetBrains/compose-multiplatform#3345]
**Describe the bug**
https://kotlinlang.slack.com/archives/C01F2HV7868/p1689140024804609

```
Uncaught Error: Error loading module 'cms'. Its dependency 'androidx-runtime' was not found. Please, check whether 'androidx-runtime' is loaded prior to 'cms'.
    at cms.js:8:13
    at cms.js:21:2
```

**Affected platforms**
Select one of the platforms below:
- Web (K/JS) - Canvas based API
- Web (K/JS) - HTML library

**Versions**
- Kotlin version*:  1.8.20
- Compose Multiplatform version*:  1.4.1

**To Reproduce**
see https://kotlinlang.slack.com/archives/C01F2HV7868/p1689140024804609

**Additional context**

**Workaround:**

Update the build.gradle.kts, by adding `commonWebpackConfig { outputFileName = ... }`
```kotlin
js(IR) {
        moduleName = "cms"
        browser {
            commonWebpackConfig {
                outputFileName = "cms.js"
            }
        }
        binaries.executable()
    }
```

TODO: create a reproducer without compose and report to k/js team.

**Comment by ZacSweers** [comment]
Hello! I filed a similar issue (that reproduces a problem described earlier in the linked slack thread) here: https://github.com/Kotlin/kotlinx-datetime/issues/334

**Comment by ZacSweers** [comment]
A similar issue was filed and marked as fixed in 1.8 here, but seems to be regressed: https://youtrack.jetbrains.com/issue/KT-45789

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-3345">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
